src/Tests/Utils/PipedSpawn.test.ts

```diff
@@ -37,6 +37,14 @@ suite('Utils', function() {
       });
     });
 
+    test('NEG: first cmd fails', function() {
+      try {
+        let cat = pipedSpawn('cat', ['invalid_file'], {}, 'grep', ['not_exist'], {});
```

Comment:
How about the following? (to show that only first line can fail and throw exception)
```suggestion
        let cat = pipedSpawn('cat', ['invalid_file'], {}, 'wc', ['-l'], {});
```